app/controllers/pages_controller.rb

```diff
@@ -52,4 +60,14 @@ def answer_types
   def convert_session_keys_to_symbols
     session[:page].deep_symbolize_keys! if session[:page].present?
   end
+
+  def setup_draft_question_for_existing_page
```

Comment:
For a future iteration - can we introduce the factory pattern to address setting up our objects outside of our controllers?